bindgen/ir/context.rs

```diff
@@ -2060,6 +2064,21 @@ If you encounter an error missing from this list, please file an issue or a PR!"
         &self.translation_unit
     }
 
+    /// Initialize fallback translation unit
+    pub(crate) fn init_fallback_translation_unit(
```

Comment:
Then this function can disappear, as `try_ensure_fallback_translation_unit` is the only thing that writes to it.